db/db_impl_write.cc

```diff
@@ -388,17 +389,6 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
   }
   if (should_exit_batch_group) {
     if (status.ok()) {
-      for (auto* writer : write_group) {
-        if (!writer->CallbackFailed() && writer->pre_release_callback) {
-          assert(writer->sequence != kMaxSequenceNumber);
-          Status ws = writer->pre_release_callback->Callback(writer->sequence,
-                                                             disable_memtable);
-          if (!ws.ok()) {
-            status = ws;
-            break;
-          }
-        }
-      }
```

Comment:
This is a little bit tricky for !status.ok() case. Now it shouldn't be a problem because it is probably turned into read-only DB. But if we are to recover from failures one day, we need to clear it up.

But I think this is better than we have now. Maybe leave a comment here, explaining the problem.